cypress.config.ts

```diff
@@ -0,0 +1,12 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Hey @e-papanicolas ! Im not sure if its good to add cypress on `dependencies` because it will be used in our production build, which its not necessary and will make our build heavier.
Cypress is a development dependency, cypress docs suggest installing it as devDep.

I was reading about this and I saw a really [good article](https://medium.com/defmethod-works/sharable-eslint-config-b96af3b93960) and an [issue](https://github.com/import-js/eslint-plugin-import/issues/422) related to this, so I added this solution, let me know what do you think about.

Reply:
@LeoFuna yeah that makes sense!